episodes/medical_imaging.md

```diff
@@ -35,17 +35,70 @@ Historically, x-rays were the first common form of medical imaging. The diagram
+
+![Floursocopy.](fig/fluoro.gif){alt='Floursocopy.'}
+*image courtesy of Ptrump16, CC BY-SA 4.0 <https://creativecommons.org/licenses/by-sa/4.0>, via Wikimedia Commons*
+
+ Flouroscopy images are stored in a DICOM but can be displayed as movies because they are typically cine-files. Cine- is a file format that lets you store images in sequence with a frame rate.
+
+
 
 ## Computed Tomography and Tomosynthesis
 
 There are several kinds of tomography. This technique produces 3D-images, made of voxels, that allow us to see structures within a subject. CTs are extremely common, and helpful for many diagnostic questions, but have certain costs in terms of not only time and money, but also radiation to patients.
 
-CTs and tomosynthetic images are produced with the same technology. The difference is that in a CT the image is based on a 360 degree capture of the signal. You can conceptualize this as a spinning donut with the generator and receptor opposite to each other. Tomosynthesis uses a limited angle instead of going all the way around the patient. In both cases, the image output is then made by processing this into a 3D-array. We see again, similarly to x-ray, what tissues attenuated the radiation, and what tissues let it pass, but we can visualize it in either 3D or single layer "slices" of voxels. It is not uncommon to get CTs as DICOM CT projection data (DICOM-CT-PD) files which can then be processed before viewing, or in some cases stored off as other file types.
+CTs and tomosynthetic images are produced with the same technology. The difference is that in a CT the image is based on a 360 degree capture of the signal. You can conceptualize this as a spinning donut with the generator and receptor opposite to each other. The raw data of a CT is a [sinogram](learners/reference.md#sinogram). Only by processing this data do we get what most people would recognize as a CT. At this level of processing there are already choices effecting the data we get.Let's examine two ways to process our sinograms:
+
+
+```python
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage.transform import iradon
+from skimage.transform import iradon_sart
+
+# load a sinogram of a simple phantom of the head
+sino = np.load('Schepp_Logan_sinogram.npy')
```

Comment:
Don't you have a data folder? Or is this code not supposed to be taught with live coding?

Reply:
If it is just to show the image, maybe it makes more sense to leave out the code here.

Reply:
This npy file will be put into the new dataset on Zenodo which  will be downloadable. That is where we are keeping things.

Reply:
In that case I still recommend to instruct participants to download those files into a `data` folder instead of the root. But it is just a minor thing.